scripts/openmc-update-inputs

```diff
@@ -1,14 +1,15 @@
 #!/usr/bin/env python
 """Update OpenMC's input XML files to the latest format.
 
-Usage information can be obtained by running 'update_inputs.py --help':
+Usage information can be obtained by running 'openmc-update-inputs --help':
 
-usage: update_lattices.py [-h] IN [IN ...]
+usage: openmc-update-inputs [-h] IN [IN ...]
 
-Update lattices in geometry.xml files to the latest format. This will remove
-'outside' attributes/elements and replace them with 'outer' attributes. Note
-that this script will not delete the given files; it will append '.original'
-to the given files and write new ones.
+Update geometry.xml files to the latest format. This will remove 'outside'
+attributes/elements from lattices and replace them with 'outer' attributes. For
+'cell' elements, any 'surfaces' attributes/elements will be renamed
+'region'. Note that this script will not delete the given files; it will append
+'.original' to the given files and write new ones.
```

Comment:
Add info about `surfaces` -> `region` to the `epilog` so that it will be shown with `--help`